src/lib/Card/index.ts

```diff
@@ -0,0 +1 @@
+export * from "./Card";
```

Comment:
make sure to also do this export from the `index.ts` file in the `src/lib` folder, so that the component can be accessible as a named export by projects installing the package